setup.py

```diff
@@ -1,5 +1,7 @@
 from setuptools import setup, find_packages
 
+with open('requirements.txt') as fp:
```

Comment:
No need for this here, the list is very simple and pretty much future proof for now, so I'd rather keep it simple as before.